frontend/src/components/SingleCarousel/index.jsx

```diff
@@ -1,17 +1,27 @@
 import React, { Component } from "react";
 import { Carousel, Navbar } from "react-bootstrap";
 import "./index.css";
-import ymim1 from "../../assets/ymim1.png";
+//import ymim1 from "../../assets/ymim1.png";
 import logo from "../../assets/logo.png";
 
 class SingleCarousel extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
this is unneeded and can be deleted as we are using props not state 

https://www.agiliq.com/blog/2018/05/understanding-react-state-and-props/

Reply:
Thanks, Dan. I removed the code.